### ISSUE TITLE
Reading: Multilingual BERT Post-Pretraining Alignment

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#179]
## 0. Paper
@inproceedings{pan-etal-2021-multilingual,
    title = "Multilingual {BERT} Post-Pretraining Alignment",
    author = "Pan, Lin  and
      Hang, Chung-Wei  and
      Qi, Haode  and
      Shah, Abhishek  and
      Potdar, Saloni  and
      Yu, Mo",
    booktitle = "Proceedings of the 2021 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies",
    month = jun,
    year = "2021",
    address = "Online",
    publisher = "Association for Computational Linguistics",
    url = "https://aclanthology.org/2021.naacl-main.20",
    doi = "10.18653/v1/2021.naacl-main.20",
    pages = "210--219",
    abstract = "We propose a simple method to align multilingual contextual embeddings as a post-pretraining step for improved cross-lingual transferability of the pretrained language models. Using parallel data, our method aligns embeddings on the word level through the recently proposed Translation Language Modeling objective as well as on the sentence level via contrastive learning and random input shuffling. We also perform sentence-level code-switching with English when finetuning on downstream tasks. On XNLI, our best model (initialized from mBERT) improves over mBERT by 4.7{\%} in the zero-shot setting and achieves comparable result to XLM for translate-train while using less than 18{\%} of the same parallel data and 31{\%} fewer model parameters. On MLQA, our model outperforms XLM-R{\_}Base, which has 57{\%} more parameters than ours.",
}

## 1. What is it?
They propose an alignment method of mBERT.

## 2. What is amazing compared to previous works?
cons of FastAlign: 
- noises lead to error propagation to the rest of the pipeline
- **word-level alignment**: contextual information is overlooked

To relax these problems, they proposed post-pretraining alignment (PPA).

## 3. Where is the key to technologies and techniques?
They propose a post-pretraining alignment (PPA).
The loss is the sum of sentence-level objective MoCo and word-level objective TLM
* TLM: extension of Masked Language Modeling. parallel sentences are concatenated
* MoCo:
<img width="457" alt="スクリーンショット 2021-07-08 0 14 41" src="https://user-images.githubusercontent.com/45454055/124785071-7fa18880-df81-11eb-905a-94ce49354844.png">

<img width="500" alt="スクリーンショット 2021-07-08 0 06 26" src="https://user-images.githubusercontent.com/45454055/124783862-57655a00-df80-11eb-850c-57ae3e8067d7.png">

## 4. How did evaluate it?
- task1 XNLI: zero-shot (train only English), translate-train (use translation data)
From Table 2, their methods achieved comparable resutls to XLM.
<img width="688" alt="スクリーンショット 2021-07-08 0 13 04" src="https://user-images.githubusercontent.com/45454055/124784839-4537eb80-df81-11eb-98af-a6584d4bcc0c.png">

- task 2 MLQA: zero-shot (train only English), translate-train (use translation data)
From Table 4, their methods also achieved high performance.
<img width="688" alt="スクリーンショット 2021-07-08 0 12 29" src="https://user-images.githubusercontent.com/45454055/124784762-305b5800-df81-11eb-9e98-482053710751.png">

## 5. Is there a discussion?
- Ablation study: From table 5, TLM is very important objective. In the zero-shot settings, MoCo is also important.
<img width="688" alt="スクリーンショット 2021-07-08 0 12 09" src="https://user-images.githubusercontent.com/45454055/124784708-25082c80-df81-11eb-8af1-c30b2b9de0a9.png">
